tox_core/src/state_format/old.rs

```diff
@@ -615,12 +614,13 @@ impl ToBytes for State {
 mod tests {
     use super::*;
 
+    use rand::thread_rng;
     use tox_packet::ip_port::*;
 
     encode_decode_test!(
         tox_crypto::crypto_init().unwrap(),
```

Comment:
Do we need crypto_init anymore?

Reply:
We have it everywhere in such tests, even when it's unneeded. I'll remove it completely in the next MR where libsodium is completely removed.